### PR TITLE
Kb for auxiliary/scanner/ftp/anonymous

### PR DESCRIPTION
**Pull request by itsmeroy2012**
### Docs for auxiliary/scanner/ftp/anonymous

### CODE REVIEW

File: documentation/modules/auxiliary/scanner/ftp/anonymous.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
+## Description
+
+This module allows us to scan through a series of IP Addresses and provide details whether anonymous access is allowed or not. By default, anonymous access is not allowed by the FTP server.
```
Comment (h00die) [inline documentation/modules/auxiliary/scanner/ftp/anonymous.md]: first sentence is missing the word FTP or FTP service, by itself it sounds like you can anonymously log in to any IP

File: documentation/modules/auxiliary/scanner/ftp/anonymous.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+## Vulnerable Application
+
+### Install ftp server on Kali Linux:
+
+1.  ```apt-get install vsftpd```
+2. Allow local users to log in and to allow ftp uploads by editing file /etc/vsftpd.conf uncommenting the following:
```
Comment (h00die) [inline documentation/modules/auxiliary/scanner/ftp/anonymous.md]: add ` around the file name

File: documentation/modules/auxiliary/scanner/ftp/anonymous.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+## Verification Steps
+
+1. Do: ```use auxiliary/scanner/ftp/anonymous```
+2. Do: ```set RHOSTS [IP]```
+3. Do: ```set RPORT [IP]```
+4. Do: ```run```
+
+## Sample Output
+
```
Comment (h00die) [inline documentation/modules/auxiliary/scanner/ftp/anonymous.md]: ### On vsFTPd 3.0.3 on Kali

File: documentation/modules/auxiliary/scanner/ftp/anonymous.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+[+] 127.0.0.1:21          - 127.0.0.1:21 - Anonymous READ (220 (vsFTPd 3.0.3))
+[*] Scanned 1 of 1 hosts (100% complete)
+[*] Auxiliary module execution completed
+msf auxiliary(anonymous) > 
+```
+
+## Confirming using NMAP
+
+```
+root@kali:~# nmap -sV -sC 127.0.0.1
```
Comment (h00die) [inline documentation/modules/auxiliary/scanner/ftp/anonymous.md]: try this with just -p 21 so it gets rid of the other stuff thats not needed

File: documentation/modules/auxiliary/scanner/ftp/anonymous.md [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+| rpcinfo: 
+|   program version   port/proto  service
+|   100000  2,3,4        111/tcp  rpcbind
+|_  100000  2,3,4        111/udp  rpcbind
+5432/tcp open  postgresql PostgreSQL DB 9.5.4
+| ssl-cert: Subject: commonName=kali
+| Not valid before: 2016-08-29T20:19:20
+|_Not valid after:  2026-08-27T20:19:20
+|_ssl-date: TLS randomness does not represent time
+Service Info: OS: Unix
```
Comment (h00die) [inline documentation/modules/auxiliary/scanner/ftp/anonymous.md]: no need for after this line of output

File: documentation/modules/auxiliary/scanner/ftp/anonymous.md [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+1. On the taskbar, click Start, point to Administrative Tools, and then click Server Manager.
+2. In the Server Manager hierarchy pane, expand Roles, and then click Web Server (IIS).
+3. In the Web Server (IIS) pane, scroll to the Role Services section, and then click Add Role Services.
+4. On the Select Role Services page of the Add Role Services Wizard, expand FTP Server.
+5. Select FTP Service. (Note: To support ASP.NET Membership or IIS Manager authentication for the FTP service, you will also need to select FTP Extensibility.)
+6. Click Next.
+7. On the Confirm Installation Selections page, click Install.
+8. On the Results page, click Close. 
+
+#### IIS 7.5 for Windows 7:
```
Comment (h00die) [inline documentation/modules/auxiliary/scanner/ftp/anonymous.md]: I dont see anything on the IIS instructions about enabling anonymous.  if its OK by default, the description up to that says its never on by default needs an update

**Comment by itsmeroy2012** [comment]
Made the changes @h00die

**Comment by h00die** [comment]
# Release Notes
adds module docs for ftp/anonymous

**Comment by h00die** [comment]
Thanks as always for the submissions!

**Pull request merged**